### PR TITLE
No wait-on in production

### PR DESCRIPTION
**Pull request by zeladada**
Also improved scripts and updated README to match.

### CODE REVIEW

**Comment by young-k** [comment]
You need to add @JunyoungLim and me on your PRs 😠

File: src/utils/GraphhopperUtils.js [hunk 1]
```diff
@@ -16,28 +16,44 @@ const waitOptions = {
...
     points_encoded: false,
     vehicle: 'pt',
+    'ch.disable': true,
     weighting: 'short_fastest',
     'pt.arrive_by': arriveBy,
-    'ch.disable': true,
     'pt.walk_speed': 3.0, // > 3.0 suggests getting off bus earlier and walk half a mile instead of waiting longer
     'pt.earliest_departure_time': getDepartureTimeDateNow(departureTimeQuery, arriveBy),
     'pt.profile': true,
     'pt.max_walk_distance_per_leg': 2000,
```
Comment (young-k) [inline src/utils/GraphhopperUtils.js]: Can we alphabetize these params?

File: scripts/run.sh [hunk 2]
```diff
@@ -73,19 +73,19 @@ RUN_TRANSIT=true
...
-        -s | --setup )          RUN_TRANSIT=false
-                                ;;
-        * )                     usage
-                                exit 1
+        -h | --help )                   usage
+                                        exit
+                                        ;;
+        -p | --prod | --production )    PROD=true
+                                        ;;
+        -d | --dev | --development )    DEV=true
```
Comment (young-k) [inline scripts/run.sh]: not the biggest fan of having two flags for `--dev` and `--development`, what's the reasoning behind not having just one?

**Comment by young-k** [comment]
@zeladada can you get to making these changes before sunday?

File: src/utils/GraphhopperUtils.js (hunk 1 above)
Comment (zeladada) [reply src/utils/GraphhopperUtils.js]: This is how they're ordered in the Graphhopper API docs

**Pull request merged**